airbyte-webapp/src/views/Connection/ConnectionForm/components/SchemaChangeBackdrop.tsx

```diff
@@ -12,7 +12,7 @@ import { OctaviaYellowFlag } from "./OctaviaYellowFlag";
 import styles from "./SchemaChangeBackdrop.module.scss";
 
 export const SchemaChangeBackdrop: React.FC<React.PropsWithChildren<unknown>> = ({ children }) => {
-  const allowAutoDetectSchemaChanges = useFeature(FeatureItem.AllowAutoDetectSchemaChanges);
+  const allowAutoDetectSchemaChanges = useFeature(FeatureItem.AllowAutoDetectSchema);
```

Comment:
```suggestion
  const allowAutoDetectSchema = useFeature(FeatureItem.AllowAutoDetectSchema);
```
to match the other spots we changed variable names